src/plone/api/content.py

```diff
@@ -187,7 +187,7 @@ def move(source=None, target=None, id=None, safe_id=False):
     # If no target is given the object is probably renamed
     if target and source.aq_parent is not target:
             target.manage_pasteObjects(
-                source.aq_parent.manage_cutObjects(source_id)
+                source.aq_parent.manage_cutObjects(source_id),
```

Comment:
against you example I like the trailing bracket on a new line. but I don't see a comma needed.